plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -736,26 +737,54 @@ public synchronized void commentTable(String databaseName, String tableName, Opt
     @Override
     public synchronized void commentColumn(String databaseName, String tableName, String columnName, Optional<String> comment)
     {
```

Comment:
"Fix COMMENT ON partitioned column for FileHiveMetastore"

The commit message states that something is fixed, but not exactly what.
Please be more specific.

https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages

Reply:
it seems to me that the message should be

"Support altering partition column comments in Hive file metastore"